go.mod

```diff
@@ -10,7 +10,7 @@ replace gopkg.in/alecthomas/kingpin.v2 => github.com/Percona-Lab/kingpin v2.2.6-
 
 replace github.com/pganalyze/pg_query_go v1.0.3 => github.com/Percona-Lab/pg_query_go v1.0.3-percona
 
-replace golang.org/x/crypto v0.1.0 => github.com/percona-lab/crypto v0.0.0-20230117105348-b57f9a698c66
+replace golang.org/x/crypto => github.com/percona-lab/crypto v0.0.0-20220811043533-d164de3c7f08
```

Comment:
no reason to depend on version, we must always override

Reply:
We can try to do it like this, but overriding exact version helped us to not forget to sync our fork with upstream. 

Also why do you changed version of our fork? It doesn't contain latest changes from upstream anymore.

Reply:
@BupycHuk I think you merged it too early.